contrib/test/instr-fixtures/bpf-loader-upgradeable-v1-programs.list

```diff
@@ -1 +1,199 @@
 dump/test-vectors/instr/fixtures/bpf-loader-upgradeable-v1-programs/instr-114U24XCn8Mxu5abKXM7XeYqETAb3wZdNAE9sRcZLqrHBM4tPDowAadcoPGg5zW75o3maq6s3N1WM4rdyw14jq-1207.fix
+dump/test-vectors/instr/fixtures/bpf-loader-upgradeable-v1-programs/058f9716ec2e93a78723d04cfbab526eeef2e448_1631513.fix
```

Comment:
These should be moved to `bpf-loader-v3.list`

Reply:
i used `-g` to group when creating fixtures and it put them under `bpf-loader-upgradeable-v1-programs`